docs/sites.yml

```diff
@@ -3611,3 +3611,15 @@
     - Education
   built_by: Michael Uloth
   built_by_url: "https://www.michaeluloth.com"
+- title: Grooovinger
```

Comment:
Looks like the linter is failing here. I think if you check this out locally, this should work to fix the error:

```bash
npx prettier --write docs/sites.yml
```

Reply:
sorry, I missed the new line. Thanks!